index.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var Promise = require('any-promise')
```

Comment:
My personal preference would be to just use native `Promise` instead of a library. Since this library uses `fs`, it's unlikely to be used in browsers where promise support might not be available, and node.js has supported promises since 4.0.

Reply:
Sounds good to me. I went this route because [Travis is running node 0.10](https://github.com/js-cli/js-empty-dir/blob/1.0.0/.travis.yml#L3), which does not have a native Promise implementation.

Reply:
np, I think if we do this we should just change the minimum travis engine to node 4. I'd still like feedback or a thumbs up/down from @tkellen on this before we publish though. 

thanks!

Reply:
Very late :+1: from me. Thanks guys!